common/metrics/metricstest/metricstest_test.go

```diff
@@ -68,3 +69,40 @@ func TestBasic(t *testing.T) {
 	require.Equal(t, float64(2), s2.MustCounter(counterName+"_total", expectedCounterTags...))
 	require.Equal(t, float64(10), s2.MustGauge(gaugeName, expectedGaugeTags...))
 }
+
+func TestHistogram(t *testing.T) {
+	logger := log.NewTestLogger()
```

Comment:
Please add a `t.Parallel()` statement on the first line of hermetic tests like this for improved performance.

Reply:
I don't see this yet, but assuming it's added then LGTM